packages/helm/symphony/files/symphony-api.json

```diff
@@ -37,8 +37,7 @@
           "host": "{{ include "symphony.redisHost" . }}",
           "requireTLS": false,
           "password": "",
-          "numberOfWorkers": 1,
-          "multiInstance": false
+          "numberOfWorkers": 20
```

Comment:
I saw DefaultNumberOfWorkers is 20. Where do we configure numberOfWorkers to 1 in standalone before we have appropriate JobId solution for standalone?